auctioneer/client.go

```diff
@@ -194,6 +201,19 @@ func getAuctionServerDialOpts(insecure bool, tlsPath string,
 		creds := credentials.NewTLS(&tls.Config{})
 		opts = append(opts, grpc.WithTransportCredentials(creds))
 	}
+	// If a SOCKS proxy address was specified, then we should dial through
+	// it.
+	if proxyAddress != "" {
+		log.Infof("Proxying connection to auction server over Tor SOCKS proxy %v",
```

Comment:
nit: lines are a bit too long if you use a tab character width of 8 spaces.

Reply:
I pushed the changes!